app/views/downloads/_download_preface.html.erb

```diff
@@ -1,24 +1,25 @@
-<p class="message"><%= ts("Posted originally on the") %> <%= link_to ArchiveConfig.APP_NAME, root_url %> 
+<p class="message"><%= ts("Posted originally on the") %> <%= link_to ArchiveConfig.APP_NAME, root_url %>
 <%= ts("at") %> <%= link_to work_url(@work), work_url(@work) %>.</p>
 
 <div class="meta">
   <dl class="tags">
     <% Tag::VISIBLE.each do |type| %>
       <% tags = @work.tag_groups[type] %>
-      <% unless tags.blank? || (type == "Warning" && hide_warnings?(@work)) || (type == "Freeform" && 
+      <% unless tags.blank? || (type == "Warning" && hide_warnings?(@work)) || (type == "Freeform" &&
 hide_freeform?(@work)) %>
         <dt><%= type.constantize::NAME %>:</dt>
         <dd><%= tags.map {|t| link_to(t.name, tag_url(t))}.join(", ").html_safe %></dd>
       <% end %>
     <% end %>
-    <% unless @work.serial_works.blank? %>
+    <% @series_list = @work.serial_works.reject{|x| x.series.nil?} %>
```

Comment:
I think `series_list` can probably be a local variable instead of an instance variable. Also, if Hound ran on views, it would say there should be spaces inside the curly braces, like so: `{ |x| x.series.nil? }` (Also, we usually use letters that relate to the thing we're handling, so maybe `sw` for serial work rather than `x`?) But this file is so messy a couple of space are definitely not the end of the world.